### ISSUE TITLE
[BUG] Dashboard sidebar is overlapped by main content elements - CSS issue

### ISSUE THREAD

**Issue by BenSturmfels** [houdiniproject/houdini#1587]
When I hover over the left-hand sidebar, it pops out, but is overlapped by various elements in the body of the page, such as the "Dashboard" header at the top of the page.

![2023-06-26_20-00](https://github.com/houdiniproject/houdini/assets/9541562/f8b11f91-923c-4447-a36e-e5bf643de14e)

This is based on the current main branch. Beforehand I ran:

```
yarn
rm -rf public/assets public/packs
bin/rails server
```

The issue appears to be that no `z-index` is set on the `.sideNav` element. Looking at the SCSS, I can see that `app/assets/stylesheets/common/z_indices.css.scss` sets a `z-index` on `.sideNav`, but this SCSS file doesn't seem to be included anywhere.

**Comment by BenSturmfels** [comment]
This also means that you can only click on the part of the menu links that was within the boundaries of the narrow menu.

**Comment by BenSturmfels** [comment]
This is probably related too - you can't create a new campaign because the modal overlay is overlapping the form:

![2023-06-26_20-50](https://github.com/houdiniproject/houdini/assets/9541562/c310766e-3c0a-4a4d-9306-c5047eb38beb)

A bunch of other forms are similarly affected.

**Comment by Abraao1231** [comment]
Hello @BenSturmfels, my colleagues and I have opened a PR that solves the problem reported in the issue. The pull request number is #1604. We would be grateful if you could review it.

**Comment by BenSturmfels** [comment]
I've just done the `git bisect` dance, which came up with the following as the likely cause:

```
5b066f655b5a41b8edf27dfc21a97b198ae97fde is the first bad commit
commit 5b066f655b5a41b8edf27dfc21a97b198ae97fde
Author: Eric Schultz <eric@wwahammy.com>
Date:   Thu Jun 22 20:16:15 2023 -0500
    Remove references to deleted stylesheets
 app/assets/stylesheets/global.css.scss                    | 1 -
 app/assets/stylesheets/nonprofits/dashboard/page.css.scss | 1 -
 2 files changed, 2 deletions(-)
```

**Comment by BenSturmfels** [comment]
> Hello @BenSturmfels, my colleagues and I have opened a PR that solves the problem reported in the issue. The pull request number is #1604. We would be grateful if you could review it.

Just noting for anyone reading that I did look at @Abraao1231's PR and make comments there (which lead me to trying `git bisect`).

**Comment by BenSturmfels** [comment]
@Abraao1231 would you like to propose a new PR based on the problematic commit above? Hint: app/assets/stylesheets/global.css.scss

**Comment by BenSturmfels** [comment]
Confirmed fixed by #1607.
